build_tools/ci/run_matmul_test.sh

```diff
@@ -659,9 +654,9 @@ run_matmul_test_on_shapes ${i32_shapes_small[@]} \
 i32_shapes_medium=(
   '1024x1024x1024'
 )
-if [ "$OSTYPE" != "msys" ]; then
-  i32_shapes_medium+=('1536x2048x1536')
-fi
+#if [ "$OSTYPE" != "msys" ]; then
```

Comment:
```suggestion
# TODO(jornt): re-enable `1536x2048x1536`
#if [ "$OSTYPE" != "msys" ]; then
```

Reply:
Or maybe better remove and create an issue for it (which I think you already had)?

Reply:
Ya https://github.com/nod-ai/iree-amd-aie/issues/701 (I know the shape there is a permutation but it's gotta be the same malfunction...)